python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
+    + \frac{1}{2} \sum_{i,j = 1}^d \left(\frac{\partial^2 h}{\partial x_i \partial x_j}\right)(\vect{\mu})
+    (X_i-\mu_i)(X_j-\mu_j) + o(||\uX||^2)
 
-| The decomposition of the variance at the order 2 is not implemented.
-  It requires both the knowledge of higher order derivatives of the model
-  and the knowledge of moments of order strictly greater than 2 of the pdf.
 
-| **Case** :math:`n_Y>1`
+If we use the first order expansion, we get:
 
+  .. math::
 
-| The perturbation approach can be developed at different orders
-  from the Taylor decomposition of the random vector :math:`\uY`. As
-  :math:`\uY=h(\uX)`, the Taylor decomposition around :math:`\ux = \muX`
-  at the second order yields to:
+       \begin{array}{lcl}
+            \Expect{Y} & \simeq & h(\vect{\mu})\\
+            \Var{Y} & \simeq & \sum_{i=1}^{d} \sigma_i^2 \left[\left(\frac{\partial h}{\partial x_i}
+            \right)(\vect{\mu}) \right]^2
+       \end{array}
 
-  .. math::
 
-      \uY = h(\muX) + <\underline{\underline{\nabla}}h(\muX) , \: \uX - \muX> + \frac{1}{2}<<\underline{\underline{\underline{\nabla }}}^2 h(\muX,\: \underline{\mu}_{\:X}),\: \uX - \muX>,\: \uX - \muX> + o(\Cov \uX)
+If we use the second order expansion, we get:
 
-where:
+  .. math::
 
--  :math:`\muX = \Expect{\uX}` is the vector of the input variables at
-   the mean values of each component.
+       \Expect{Y}  \simeq h (\vect{\mu}) + \frac{1}{2} \sum_{i,j=1}^{d} c_{ij}\left(\frac{\partial^2 h}
+       {\partial x_i \partial x_j}\right)(\vect{\mu})
 
--  :math:`\Cov \uX` is the covariance matrix of the random vector
-   :math:`\uX`. The elements are the followings :
-   :math:`(\Cov \uX)_{ij} = \Expect{\left(X^i - \Expect{X^i} \right)^2}`
 
--  :math:`\underline{\underline{\nabla}} h(\muX) = \: ^t \left( \frac{\partial y^i}{\partial x^j}\right)_{\ux\: =\: \muX} = \: ^t \left( \frac{\partial h^i(\ux)}{\partial x^j}\right)_{\ux\: =\: \muX}`
-   is the transposed Jacobian matrix with :math:`i=1,\ldots,n_Y` and
-   :math:`j=1,\ldots,n_X`.
+The second order approximation of the variance  is not implemented because it requires both the knowledge of higher
+order derivatives of :math:`h` and the knowledge of moments of order strictly greater
+than 2 of the distribution of :math:`\uX`.
 
--  :math:`\underline{\underline{\underline{\nabla^2}}} h(\ux\:,\ux)` is
-   a tensor of order 3. It is composed by the second order derivative
-   towards the :math:`i^\textrm{th}` and :math:`j^\textrm{th}`
-   components of :math:`\ux` of the :math:`k^\textrm{th}` component of
-   the output vector :math:`h(\ux)`. It yields to:
-   :math:`\left( \nabla^2 h(\ux) \right)_{ijk} = \frac{\partial^2 (h^k(\ux))}{\partial x^i \partial x^j}`
 
--  :math:`<\underline{\underline{\nabla}}h(\muX) , \: \uX - \muX> = \sum_{j=1}^{n_X} \left( \frac{\partial {\uy}}{\partial {x^j}}\right)_{\ux = \muX} . \left( X^j-\muX^j \right)`
+**Case 2:** :math:`Y =(Y_1, \dots, Y_q) \in \Rset^q` with :math:`q>1`:
```

Comment:
Instead of using a bold face text, please create a new subsection.

Reply:
ok